terminus-ui/autocomplete/src/autocomplete.component.spec.ts

```diff
@@ -61,652 +61,4 @@ describe(`TsAutocompleteComponent`, function() {
     expect(fixture.debugElement.query(By.css('.ts-autocomplete'))).toBeTruthy();
   });
 
```

Comment:
Are these tests not needed?
How is removing them related to the change from codecov to codacy?

Reply:
We talked in person, but putting here for our future selves - removed tests to try to get coverage to drop during Codacy testing. I will undo this before merging